hack/update/update_all/update_all.go

```diff
@@ -38,10 +38,11 @@ func main() {
 
 	var changes []string
 	i := 0
+	max := 100
```

Comment:
Should be a constant but I don't think we need it.

Instead we should have a summary at the end like:

    updated: 42, skipped: 7, failed: 1